src/Core/src/Platform/Android/GraphicsViewExtensions.cs

```diff
@@ -8,5 +9,10 @@ public static void UpdateDrawable(this PlatformGraphicsView PlatformGraphicsView
 		{
 			PlatformGraphicsView.Drawable = graphicsView.Drawable;
 		}
+
+		internal static void UpdateBackground(this PlatformGraphicsView PlatformGraphicsView, IGraphicsView graphicsView)
+		{
+			PlatformGraphicsView.BackgroundColor = graphicsView.Background?.ToColor() ?? Colors.White;
```

Comment:
White or Transparent? What was it before if you don't set a color? What is the "default" color?

Reply:
Yea, my bad... The default was, so I shouldn't have changed it to white😅